packages/ui/src/components/va-alert/VaAlert.stories.ts

```diff
@@ -1,12 +1,217 @@
-import { defineComponent } from 'vue'
-import VaAlert from './VaAlert.demo.vue'
+import { VaAlert } from './'
+import { VaButton } from '../va-button'
+import { userEvent } from '../../../.storybook/interaction-utils/userEvent'
+import { within } from '@storybook/testing-library'
+import { expect } from '@storybook/jest'
 
 export default {
   title: 'VaAlert',
   component: VaAlert,
+  tags: ['autodocs'],
 }
 
-export const Default = defineComponent({
+export const Default = {
+  args: {
+    description: 'Message',
+  },
+}
+
+export const Dense = {
+  args: {
+    dense: true,
+    description: 'Message',
+  },
+}
+
+export const Title = {
+  args: {
+    title: 'Title',
+    description: 'Message',
+  },
+}
+
+export const Center = {
```

Comment:
For center we also need component with icon and close, something like this (ignore colors and messages, both should have title and description):

<img width="837" alt="image" src="https://github.com/epicmaxco/vuestic-ui/assets/5394573/1867c8d7-0548-4b66-af65-1de3a8222e1d">